internal/pkg/client/cache/dir.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// Copyright (c) 2017-2019, Sylabs Inc. All rights reserved.
```

Comment:
I *think* this file should be `2018-2019`, since it didn't exist in 2017. @jmstover can you make sure that's correct